sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -28,8 +28,8 @@ _Azure SDK Python packages support for Python 2.7 has ended 01 January 2022. For
 
 #### Create a Cognitive Services or Language resource
```

Comment:
idk if it is me translating, but `Language resource` sounds weird and when I look at the Portal it says Language Service.
What about saying `Create a Cognitive Services or a Language Service resource`

Reply:
If you like it, this will apply to other places

Reply:
The service docs do use "Language resource" or "Azure Language resource" but I don't love it either. Will look at updating to Language service resource